portfolio-project-matching-app/components/ProjectCard.js

```diff
@@ -19,8 +20,8 @@ const ProjectCard = ({ project }) => {
             <div className='flex flex-wrap'>
                 {project.technologies.map((technology) => {
                     return (
-                        <div className='py-1 pr-2'>
-                            <Button key={technology.id} text={technology.name}/>
+                        <div key={technology.id} className='py-1 pr-2'>
```

Comment:
Same here, I can't take credit for this change. I simply brought in Charlie's fix from earlier today. Not sure why this is showing up on here.. I would have thought Sprint-3 would show this as already changed.